e2e-playwright/README.md

```diff
@@ -0,0 +1,66 @@
+## Quick Start
+
+To start the development, run:
+
+```
+npm install
+npm run install:browsers (only for the first time)
+npm run test (or npm run test:chrome for browser specific)
+```
+
+where `new-project` is your desired folder. This should result in cloning the project, setting up your repository as
```

Comment:
I think this sentence has no beginning 😅

Reply:
also this paragraph describes cloning, which is not included in this readme

Reply:
Yeah i copy-paste part of root README.md and i forgot to remove it. :D TY